cookbooks/travis_build_environment/recipes/pip.rb

```diff
@@ -10,6 +10,8 @@
   cwd Chef::Config[:file_cache_path]
   code <<-INSTALL_PIP
     python3 get-pip.py
+    python2 get-pip.py
+    python get-pip.py
```

Comment:
These will break in Focal (it only provides `python3`, with an optional package symlinking `python` to it) but are still needed for images for older releases.

We can fix that when we get there though (in particular, we don't know if Travis will be providing the optional symlink), so not insisting on addressing this in this PR...